plugins/modules/gitlab_user.py

```diff
@@ -346,19 +346,19 @@ def get_user_id(self, user):
 
     '''
     @param user User object
-    @param sshkey_name Name of the ssh key
+    @param sshkey_key Public key of the ssh key
     '''
-    def ssh_key_exists(self, user, sshkey_name):
-        keyList = map(lambda k: k.title, user.keys.list(all=True))
+    def ssh_key_exists(self, user, sshkey_key):
+        keySet = {ssh_key_value(k.key) for k in user.keys.list(all=True)}
```

Comment:
You likely need to use something like
```suggestion
        keySet = set(ssh_key_value(k.key) for k in user.keys.list(all=True))
```
so that this is valid Python 2.6 syntax.

Reply:
Thanks for the hint. I didn't know this needed python 2.6 compliance. This is fixed now.